src/stan-users-guide/simple_linear_regression/regression_efficient.stan

```diff
@@ -0,0 +1,19 @@
+data {
+  int<lower=0> N;
+  vector[N] x;
+  vector[N] y;
+}
+
+parameters {
+  real alpha_hat;
+  real beta_hat;
+  real<lower=0> sigma_hat;
+}
+
+//Just vectorized right now. 
+model {
+  alpha_hat ~ normal(5,10);
```

Comment:
@andrewgelman suggested making these all unit scale because we're just simulating data.  So that'd be
`normal(0, 1)` for all these.

Reply:
Don't I have to unit scale the data then?

Reply:
Yes, the data should be generated from the prior.   There's no way to generate the predictors from the prior because there's no model there.  So the usual approach is to consider data sets with independent predictors or ones with strongly correated predictors (by drawing from a high-correlation multinormal).

Reply:
I could use review on the multi-predictor case for 
docs/src/stan-users-guide/example_code/simple_linear_regression/run.R line 93 onwards. 

I am not sure I am generating the simulated data correctly.

thanks

Reply:
This is in the footnotes branch. I am trying to get it merged.

Reply:
I am going to resolve this conversation just to try and get the changes merged. Happy to revisit.